qt/CMakeLists.txt

```diff
@@ -118,9 +118,9 @@ copy_resources(
   colors.txt
   copyright.html
   countries.txt
-  drules_proto_clear.bin
-  drules_proto_dark.bin
-  drules_proto_vehicle_clear.bin
+  drules_proto_default_light.bin
+  drules_proto_default_dark.bin
+  drules_proto_vehicle_light.bin
```

Comment:
hmm why there is no outdoors style here?

Reply:
I would guess because the desktop application has no outdoors style?

Reply:
actually it has, but there is no UI for switching;
one has to type `?olight` in the search - it works well